src/tour.gleam

```diff
@@ -585,3 +616,34 @@ fn lesson_html(page: Lesson) -> String {
   |> htmb.render_page("html")
   |> string_builder.to_string
 }
+
+// This script is inlined in the response to avoid FOUC when applying the theme
+const theme_picker_js = "
+function setDarkTheme() {
+  document.documentElement.classList.add('theme-dark')
+  document.documentElement.classList.remove('theme-light')
+  localStorage.setItem('theme', 'dark')
+};
+
+function setLightTheme() {
+  document.documentElement.classList.add('theme-light')
+  document.documentElement.classList.remove('theme-dark')
+  localStorage.setItem('theme', 'light')
+};
+
+// Add handlers for theme selection buttons
+document.querySelector('.js-theme-button-light').addEventListener('click', (e) => {
+  setLightTheme()
+})
+document.querySelector('.js-theme-button-dark').addEventListener('click', (e) => {
+  setDarkTheme()
+})
+
+// Apply the preferred theme
+const theme = localStorage.getItem('theme') || 'light'
+if (theme == 'dark') {
+  setDarkTheme()
+} else {
+  setLightTheme()
+}
+"
```

Comment:
This can be simplified a bit if we use only one class

```js
const classes = document.documentElement.classList
const isDark = localStorage.getItem('dark-theme') || false
if (isDark) classes.add('theme-dark')

document.querySelector('[data-theme-toggle]').addEventListener('click', () => {
  const on = classes.toggle('theme-dark')
  localStorage.setItem('theme-dark', on)
})
```

Reply:
done! indeed it simplifies things a lot